lochness/email/__init__.py

```diff
@@ -156,6 +156,21 @@ def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
                                   'Protected', 'Study', 'Processed', 'Subject',
                                   'Datatype', 'Download time (UTC)']
         s3_df_selected.reset_index().drop('index', axis=1, inplace=True)
+
+        # too many dicom file names -> remove
+        subject_mri_gb = s3_df_selected[
+                s3_df_selected.Datatype == 'mri'].groupby('subject')
+
+        sample_mri_df = pd.DataFrame()
+        for _, table in subject_mri_gb:
+            subject_mri_sample = table.iloc[:2]  # select only two raws
+            subject_mri_sample.iloc[1]['File name'] = '...'
+            sample_mri_df = pd.concat([sample_mri_df, subject_mri_sample])
+
+        s3_df_selected = pd.concat([
+            s3_df_selected[s3_df_selected.Datatype != 'mri'],
+            sample_mri_df]).sort_index()
```

Comment:
After `group_by()`, I have only used `reset_index()`. Can you tell briefly why it is `sort_index()`?

Reply:
I'm adding `.sort_index()` for the final dataframe have the original order to rows. Without this, all the mri rows will be attached at the end of the table, which I do not want.